src/components/general/Navbar/Notifica/Notifications/items/EntryReview/index.tsx

```diff
@@ -0,0 +1,212 @@
+import React from 'react';
+import { reverseRoute } from '@togglecorp/fujs';
+
+import DisplayPicture from '#components/viewer/DisplayPicture';
+import { pathNames } from '#constants';
+import _ts from '#ts';
+
+import Notification from '../Notification';
+import LinkItem from '../LinkItem';
+
+type NotificationStatus = 'seen' | 'unseen';
+
+const REVIEW_TYPE_COMMENT = 0;
+const REVIEW_TYPE_APPROVE = 1;
+const REVIEW_TYPE_UNAPPROVE = 2;
+const REVIEW_TYPE_VERIFY = 3;
+const REVIEW_TYPE_UNVERIFY = 4;
+
+type ReviewType = 0 | 1 | 2 | 3 | 4;
+
+interface EntryReviewProps {
+    notificationType: string;
+    notification: {
+        id: number;
+        project: number;
+        receiver: 5;
+        status: NotificationStatus;
+        timestamp: string;
+        data: {
+            id: number;
+            commentType: ReviewType;
+            lead: number;
+            entry: number;
+            text: string;
+            createdByDetails: {
+                id: number;
+                name: string;
+                email: string;
+            };
+            projectDetails: {
+                id: number;
+                title: string;
+            };
+        };
+    };
```

Comment:
Please separate this into a separate type

Reply:
Done!